SECURITY.md

```diff
@@ -0,0 +1,93 @@
+# Security Policy
+
+1. [Reporting security problems](#reporting)
+4. [Security Bug Bounties](#bounty)
+2. [Incident Response Process](#process)
+
+<a name="reporting"></a>
+## Reporting security problems to Solana
+
+**DO NOT CREATE AN ISSUE** to report a security problem. Instead, please send an
+email to security@solana.com and provide your github username so we can add you
+to a new draft security advisory for further discussion.
+
+Expect a response as fast as possible, within one business day at the latest.
+
+<a name="bounty"></a>
+## Security Bug Bounties
+We offer bounties for critical security issues.  Please see [Bug Bounty
+Compensation](https://forums.solana.com/t/tour-de-sol-updates-to-tour-de-sol-and-bug-bounty-compensation-structure/1132)
+for more details.
+
+<a name="process"></a>
+## Incident Response Process
+
+In case an incident is discovered or reported, the following process will be
+followed to contain, respond and remediate:
+
+### 1. Establish a new draft security advisory
+In response to an email to security@solana.com, a project administrator will create a
+new draft security advisory for the incident at
+https://github.com/solana-labs/solana/security/advisories with the available
+information.
+
+By default only the core development team and incident reporter will have access
+to the draft security advisory.  Other relevant third parties may be added but
+access should be kept to a minimum until public disclosure.
+
+### 2. Triage
+Within the draft security advisory, discuss and determine the severity of the
+issue.
+
+If this not a critical network issue, such as a possible loss of availability or
+funds, then revert to the normal Solana github issue workflow.
+
+### 3. Prepare Fixes
+For the affected branches, typically all three (edge, beta and stable), prepare
+a fix for the issue and push them to the corresponding branch in the private
+repository associated with the draft security advisory.
+
+There is no CI available in the private repository so you must build from source
+and manually verify fixes.
+
+Code review from the incident reporter is ideal, as well as from multiple
+members of the core development team.
+
+Once the fix is accepted, prepare a single patch file for each affected branch.
+The commit title for the patch should only contain the advisory id, and not
+disclose any further details about the incident.
+
+Copy the patches to https://release.solana.com/ under a subdirectory named after
```

Comment:
How?  Or does this require privileged info?

Reply:
yeah, you need non-public info to access the AWS bucket.   I added a bit more info about that in the .md